build.gradle

```diff
@@ -27,7 +29,7 @@ subprojects{
     afterEvaluate {project ->
         if(project.hasProperty("dependencies")){
             dependencies {
-                compile 'com.esri.arcgisruntime:arcgis-android:100.0.0-beta-2'
+                compile 'com.esri.arcgisruntime:arcgis-android:100.0.0-1334'
```

Comment:
Yes, I have updated the README to reflect the public url and public release version this active branch will be working from when released.